content/terraform/v1.13.x/docs/language/block/import.mdx

```diff
@@ -6,6 +6,12 @@ description: >-
 
 # `import` block reference
 
+<Note>
+
+The beta release of Terraform 1.14 lets you configure queries to search your existing infrastructure for resources and generate configuration to import them. Refer to the [Import existing resources in bulk](/terraform/language/v1.14.x/import/bulk) beta documentation for more information.
```

Comment:
```suggestion
Terraform `v1.14.x (beta)` lets you configure queries to search your existing infrastructure for resources and generate configuration to import them. Refer to the [Import existing resources in bulk](/terraform/language/v1.14.x/import/bulk) beta documentation for more information.
```
Should we link to the `list` block reference instead of the main usage page? I would assume that it's probably a different user who might be more interested in the language construct. Your call.